### DISCUSSION TITLE
How can I use a common variable and a client variable without a prefix? Nuxt 3

### DISCUSSION THREAD

**Discussion by fenn1ks** [t3-oss/t3-env#308] [Q&A]
How can I use a common variable and a client variable without a prefix?

**Comment by juliusmarminge** [comment]
what's the usecase? this limitation is there cause most frameworks will not expose vars to the client that doesn't have a prefix. Note however that client vars are most likely also avaialbel on the server so `NEXT_PUBLIC_FOO` for example can be accessed by both environments
